.travis.yml

```diff
@@ -29,15 +29,6 @@ env:
 # so it doesn't need to be re-defined below.
 matrix:
   include:
-  # The versions listed below run within Precise distro with PHP 5.2 due to Travis limitation
-  # Hard-coding the versions since WP 5.2 has a PHP 5.6 minimum support.
-  # When removing this, remove corresponding setup functions in setup-travis.sh.
-  - php: "5.2"
-    env: WP_MODE=single WP_BRANCH=5.1
```

Comment:
Seems like we'll still need 5.1 for now as we support the previous version of WP too.